taxi/tests/test_views.py

```diff
@@ -83,6 +92,13 @@ def test_car_detail_retrieve(self):
 
         self.assertEqual(response.status_code, 200)
 
+    def test_car_search_test(self):
+        response = self.client.get(CAR_LIST_URL + "?model=e")
+        response_res = list(response.context["car_list"])
```

Comment:
please, fix everywhere

Reply:
I choose a different path. Can it be so?